website/docs/d/lb_service_monitor.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_service_monitor"
+sidebar_current: "docs-vcd-data-source-lb-service-monitor"
+description: |-
+  Provides a NSX load balancer service monitor data source.
+---
+
+# vcd\_lb\_service\_monitor
```

Comment:
maybe add `data`

Reply:
I think it is not needed as the `data` is more of terraform keyword like `resource` and website menu clearly separates these objects.

Reply:
cc: @lvirbalas

Reply:
Sorry, don't get where you're suggesting to put `data`. Looking at the latest doc version it looks very nice, but I'd like to ask one addition to example (I'll add the ask to the corresponding line).

Reply:
Different providers do it differently. vSphere uses same "silent" approach as us, AWS to the contrary has "Data Source: " or "Resource: " before each title.

Reply:
Agreed to go with naked names so far.